docs/api/vector.md

```diff
@@ -67,6 +67,7 @@ vector/utils
 ```{eval-rst}
 .. autoproperty:: gymnasium.vector.VectorEnv.unwrapped
 .. autoproperty:: gymnasium.vector.VectorEnv.np_random
+.. autoproperty:: gymnasium.vector.VectorEnv.np_random_seed
```

Comment:
I just realised we haven't actually add this to vector currently as `VectorEnv` has been separated from `Env`. We could do add using an array

Reply:
I'm not sure what you mean. Currentlly, the reset of the VectorEnv only supports a single seed (contrary to what the typing and docstring are suggesting). I could fix that, but I'd rather do it in a separate PR, b/c it's not directly related to the persistence of the seed

Reply:
Then the docstring and type hint for seed in `VectorEnv` should be adjusted (in a separate PR). Regarding the sync-async: does that require changes in this PR? Could you point me to the right env class that needs extension?

Reply:
Good point that should be updated. 
Look at the `gymnasium/vector/async_vector_env` and `.../sync_vector_env`